Project/webpack.config.js

```diff
@@ -17,7 +21,7 @@ const PORT = process.env.port || 8080;
 
 
 const oneSignalRegistrationTokenToAcsUserAccesTokenMap = new Map();
-const registerCommunicationUserForOneSignal = async (communicationUserToken) => {
+const registerCommunicationUserForOneSignal = async (communicationUserToken, userId) => {
```

Comment:
same reason as below

Reply:
It should be (communicationAccessToken, communicationUserIdentifier) just for cleanes.

Since these are things coming from an ACS lib, it would be nice to make the var names clearer